### DISCUSSION TITLE
I need help configuring axios to use a proxy

### DISCUSSION THREAD

**Discussion by DonBranson** [axios/axios#4558] [General]
This is for development purposes, in order to capture React Native requests in JMeter. We've tried configuring the proxy to point at JMeter, but the app goes straight the the backend, bypassing the proxy. What am I doing wrong? The backend server is on 8080, JMeter on 8000. Our app runs in an Android emulator or an iOS simulator. Here's our create:

```
const instance = axios.create({
  proxy: {
    protocol: 'http',
    host: '192.168.1.142',
    port: 8000,
  },
});
```

The I address is one from my workstation; I've tried `localhost` and `127.0.0.1` also.

**Comment by vrishtrix** [comment]
This has already been solved in [Issue #2072](https://github.com/axios/axios/issues/2072#issuecomment-567473812). If however, this does not fix the issue, you can create a new issue from discussion.
